tensorboard/webapp/metrics/views/right_pane/settings_view_component.ts

```diff
@@ -65,6 +65,7 @@ export class SettingsViewComponent {
 
   @Input() isCardWidthSettingEnabled!: boolean;
   @Input() isLinkedTimeFeatureEnabled!: boolean;
+  @Input() isScalarStepSelectorFeatureEnabled!: boolean;
   @Input() isScalarStepSelectorEnabled!: boolean;
   @Input() selectTimeEnabled!: boolean;
```

Comment:
selectTimeEnabled => isLinkedTimeEnabled?

So you have isLinkedTimeFeatureEnabled and isLinkedTimeEnabled just like for ScalarStepSelector. This one might be worth doing in this PR but up to you.

Reply:
already started on the giant renaming PR before seeing this comment. Will put in following PR